molecule/kerberos-rhel/molecule.yml

```diff
@@ -22,7 +22,7 @@ platforms:
     groups:
       - ${CONTROLLER_HOSTGROUP:-zookeeper}
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile-rhel-java17.j2
+    dockerfile: ../Dockerfile-rhel7-java17.j2
```

Comment:
why did we update this? and where are we using dockerfile-rhel-java17? I still see that file in 7.5.x

Reply:
`dockerfile-rhel-java17` is used for cento8 images. So removed it from here. Or else that too would need the change in mirror files for yum